src/frontend/src/lib/types/qr-code.ts

```diff
@@ -6,3 +6,28 @@ export type QrResponse = {
 	token?: string;
 	amount?: number;
 };
+
+type DecodedUrnBase = {
+	prefix: string;
+	destination: string;
+	networkId?: string;
+	functionName?: string;
+};
+
+export const urnNumericParams = ['amount', 'value'] as const;
```

Comment:
I would expose constant in capital letters for consistancy wdyt?

`URN_NUMERIC_PARAMS`